src/components/card/card.jsx

```diff
@@ -12,11 +12,23 @@ import * as styles from "./card.module.scss";
 const Card = ({ data }) => {
   const categoryBlog = useCategory(data.frontmatter.category);
 
+  const urlImages = () => {
+    if (typeof window !== 'undefined' && `${window.location.href}/` === window.location.origin) {
```

Comment:
I'm not sure about this condition, is `${window.location.href}/` === window.location.origin ok? What do you try to verify?

Reply:
I am trying to check if my href is the same as the origin, the origin is just the localhost:8000 and the href is localhost:8000/blabla/blabla If they are the same so my url is data.frontmatter.thumbnail, but if they are different I need to put the all url

Reply:
The slash is not required.